themes/classic/_dev/js/product.js

```diff
@@ -6,6 +6,51 @@ $(document).ready(function () {
   coverImage();
   imageScrollBox();
 
+  $('body').on(
```

Comment:
This code has been removed here: https://github.com/PrestaShop/PrestaShop/pull/6605
I think you have failed a rebase 😉